### DISCUSSION TITLE
InputText not working in FullScreen Mode

### DISCUSSION THREAD

**Discussion by jcoppage** [rexrainbow/phaser3-rex-notes#210] [Q&A]
Hi Rex, love the InputText plugins.  Question: my InputText object is not showing up when in FullScreen mode.  It appears when in not full screen mode. I added:  fullScreenTarget: document.getElementById('ChatInputText'), in my config but still nothing.  I'm guessing that Dom elements are hidden when going into Full Screen.   Any ideas? Thanks?
<img width="602" alt="Screen Shot 2021-12-07 at 10 19 37 PM" src="https://user-images.githubusercontent.com/70357667/145158506-0e6263af-370f-45f3-9eae-a4e20e430f7b.png">
<img width="428" alt="Screen Shot 2021-12-07 at 9 12 59 PM" src="https://user-images.githubusercontent.com/70357667/145158511-1b3182fd-be46-473b-98f6-bc29d33c353b.png">
<img width="443" alt="Screen Shot 2021-12-07 at 9 12 04 PM" src="https://user-images.githubusercontent.com/70357667/145158515-731a2cf8-67a7-4c07-b553-b649e8ca1512.png">

UPDATE EDIT: I added:" fullScreenTarget: document.getElementById('ChatInputText') " under the create() method in my UIScene.js, the Input Text object shows up! But now the object is literally the full screen :/. I'll keep updating for reference

Jeff

**Comment by jcoppage** [comment]
UPDATE EDIT: I added:" fullScreenTarget: document.getElementById('ChatInputText') " under the create() method in my UIScene.js, the Input Text object shows up! But now the object is literally the full screen :/. I'll keep updating for reference

**Comment by rexrainbow** [comment]
It seems to be a bug of phaser3 dom game object. I had opened an [issue](https://github.com/photonstorm/phaser/issues/5940) for it.

**Comment by rexrainbow** [comment]
The key is set 'config.fullscreenTarget' to parent id. See this [test code](https://codepen.io/rexrainbow/pen/ZEXOENR), line 45.

**Reply by jcoppage** [reply]
Awesome, that helped me fix it.  Thank you.  I'm guessing because my game is set up a bit differently and it has multiple scenes, the exact way in your example didn't exactly work for my particular game, but for reference this is how I got it to work: I added:  ``` this.scale.fullscreenTarget = document.getElementById('phaser-game'); ``` in the create() method in my UIScene.js scene file (where the InputText object is created). ``` Parent: 'phaser-game' ``` is located in the scale{} block in my config object which is my index.js file. 

As always thanks Rex!
Jeff
